core/modules/wiki.js

```diff
@@ -696,21 +696,44 @@ exports.getTiddlerAsJson = function(title) {
 	}
 };
 
-exports.getTiddlersAsJson = function(filter,spaces) {
+/*
+Options:
+spaces: number of spaces
```

Comment:
According to `JSON.stringify(value, replacer, space)` ... space can be a **number** or a **string**. IMO we should use this possibility to reduce file-size but keeping the JSON readable, if needed.

Reply:
The current implementation doesn't include whitespace in the JSON representation.